src/EthPool.sol

```diff
@@ -59,21 +55,26 @@ contract EthPool is Pool {
         uint64 lastUpdate,
         uint128 lastBalance,
         Receiver[] calldata currReceivers,
-        uint128 withdraw,
+        uint128 reduceBalance,
```

Comment:
Okay for the ERC20 `balanceDelta` we have positive is `topUp` for the EthPool we change it to a `uint` and call it reduceBalance where only positive is possible and positive means withdraw.